editor/editor_inspector.cpp

```diff
@@ -3250,6 +3250,7 @@ void EditorInspector::_edit_set(const String &p_name, const Variant &p_value, bo
 					undo_redo->add_undo_method(r, "setup_local_to_scene");
 				}
 			}
+			r->emit_signal("changed");
```

Comment:
This should go through `undo_redo` like the rest, and can likely use `emit_changed` directly.

Note: I don't know if that solution is correct/acceptable, but if it is then the above applies.

Reply:
Yep I was able to call the protected method this way! Thanks!